src/theme/foundations/global.ts

```diff
@@ -48,6 +48,9 @@ const global = {
   "*, *::before, &::after": {
     boxSizing: "inherit",
   },
+  ".chakra-modal__content-container": {
+    zIndex: "999999 !important",
+  },
```

Comment:
The button within the `MultiSelect` has a z-index of 10000, but it is showing above the `FeedbackBox` background.  It seems that the z-index of 1400 that is applied to the `FeedbackBox` popup element (the `role="dialog"` element)  is what is causing the problem.  And that value is coming from the `--nypl-zIndices-modal` CSS var.

<img width="605" alt="image" src="https://github.com/user-attachments/assets/28e38b98-48ab-48db-90ea-61142998b1fa">